common/tier4_state_rviz_plugin/src/include/velocity_steering_factors_panel.hpp

```diff
@@ -49,6 +50,14 @@ class VelocitySteeringFactorsPanel : public rviz_common::Panel
   void onInitialize() override;
 
 protected:
+  static constexpr double JERK_MIN = 0.0;
+  static constexpr double JERK_DEFAULT = 1.0;
+  static constexpr double JERK_MAX = 2.0;
```

Comment:
I guess we don't need max value for spin box as well.

Reply:
Sorry, I was wrong.Minimum value is actually necessary :pray: 
So, I think the range will be 0.0 to inf.

Reply:
What do you think?

Reply:
I agree the minimum value to be 0.0.
I leave the maximum as the default 99.99.